evap/staff/templates/staff_user_merge.html

```diff
@@ -11,308 +11,311 @@
 {% block content %}
     {{ block.super }}
     <h3>{% trans 'Merge users' %}</h3>
```

Comment:
This looks buggy on firefox 57:
![grafik](https://user-images.githubusercontent.com/13838962/34800600-d1aaeb80-f664-11e7-8061-1c9d5b857037.png)

Reply:
resolutions below 1200px are currently not fully supported. thoughts went only into contributor and student pages so far. would be part of #883.

Reply:
Hmm, strange, I'm pretty sure I had my browser close to 1920px wide when I took the screenshot - the screenshot is ~1400px wide, which is exactly how wide the merge cardbox is on my screen when I have the browser open fullscreen. I also counter checked against the current master, the page looked fine with that on the same browser window.

But: I just tried to replicate that and couldn't, so I guess, I messed up somewhere. #833 it is, then.